skiko/src/awtMain/kotlin/org/jetbrains/skiko/HardwareLayer.kt

```diff
@@ -110,7 +109,7 @@ internal open class HardwareLayer(
         // and its accessibility context. This timeout is used to deal with concurrency
         // TODO Find more reliable procedure
         resetFocusAccessibleJob?.cancel()
-        resetFocusAccessibleJob = GlobalScope.launch(Dispatchers.Swing) {
+        resetFocusAccessibleJob = GlobalScope.launch(MainUIDispatcher) {
```

Comment:
Maybe we rename `MainUIDispatcher` to `Dispatchers.SkikoUI`?

Reply:
Well, it's hard question, as `Dispatchers.XXX` assumes that it's part of coroutines library. Guess we'd better keep as is.

Reply:
Ok